src/components/NotificationProvider/messageBuilder.tsx

```diff
@@ -0,0 +1,49 @@
+import {
+  NotificationAction,
+  NotificationType,
+  QueuedNotification,
+} from "./types";
+import React, { ReactNode } from "react";
+
+export const queue = (notification: NotificationType): QueuedNotification => {
+  return { state: { queuedNotification: notification } };
+};
+
+export const info = (
+  message: string | ReactNode,
+  title?: string
+): NotificationType => {
+  return {
+    message,
+    title,
+    type: "information",
```

Comment:
I think we can avoid hard-coded values here, and reference the value from the `NotificationSeverity` enum (to be imported):
```suggestion
    type: NotificationSeverity.INFORMATION,
```
And similarly for the other methods below.